src/remoteConnector.ts

```diff
@@ -440,8 +441,9 @@ export default class RemoteConnector extends Disposable {
 
 		const ownerToken = await withServerApi(session.accessToken, serviceUrl.toString(), service => service.server.getOwnerToken(workspaceId), this.logger);
 
+		let password: string | undefined = ownerToken;
```

Comment:
@jeanp413 Should not we change our ssh check too? Does it have any value if it does not emulate how SSH Remote connects?

Reply:
Check is useful if user is being blocker by a firewall and for validating the host key, we emulate how ssh remote connects by connecting to the ssh agent

Reply:
vscode docs saya agent runs by default on mac https://code.visualstudio.com/docs/remote/troubleshooting#_setting-up-the-ssh-agent